cmd/bosun/expr/funcs.go

```diff
@@ -355,7 +367,7 @@ func parseGraphiteResponse(req *graphite.Request, s *graphite.Response, formatTa
 		// build tag set
 		nodes := strings.Split(res.Target, ".")
 		if len(nodes) < len(formatTags) {
-			return nil, fmt.Errorf(`returned target "%s" does not match format "%s"`, res.Target, formatTags)
+			return nil, graphite.NewParseError(fmt.Sprintf("returned target '%s' does not match format '%s'", res.Target, strings.Join(formatTags, ",")), nil, req.Url)
```

Comment:
I cannot see any reason in any uses of NewParseError why we wouldn't just use fmt.Errorf instead. The purpose here was to have varying levels of display in the log and the dashboard. I don't think this is needed for that.